docker-compose-full.yml

```diff
@@ -1,31 +1,53 @@
-version: "3.2"
+version: "2.4"
```

Comment:
Why has this downgraded?

Reply:
To allow fined control over the processes, and is generally better for docker-compose uses.  If someone wants to do swarm or something more complex than can bump the version manually.

Reply:
```suggestion
# Version 2 allows for more fine control over the processes (memory limits, etc) than version 3.
version: "2.4"
```